Keas.Mvc/ClientApp/components/Keys/KeyTable.tsx

```diff
@@ -27,6 +26,134 @@ interface IRow {
 export default class KeyTable extends React.Component<IProps, {}> {
   public render() {
     const { filters, keysInfo } = this.props;
+    const columns = [
+      {
+        Cell: (row: IRow) => (
+          <Button
+            color='link'
+            onClick={() => this.props.showDetails(row.original.key)}
+          >
+            Details
+          </Button>
+        ),
+        Header: 'Key',
+        className: 'key-details',
+        filterable: false,
+        headerClassName: 'key-details',
+        maxWidth: 150,
+        resizable: false,
+        sortable: false
+      },
+      {
+        Header: 'Key Name',
+        accessor: 'key.name',
+        className: 'word-wrap',
+        filterMethod: (filter: IFilter, row: IRow) =>
+          !!row[filter.id] &&
+          row[filter.id].toLowerCase().includes(filter.value.toLowerCase())
+      },
+      {
+        Header: 'Key Code',
+        accessor: 'key.code',
+        filterMethod: (filter: IFilter, row: IRow) =>
```

Comment:
I have a feeling none of these are doing anything, but it's worth checking.  I don't think any of this code is being called though, so compare the old version on test to your new version to make sure the filtering is the same.